src/app/recipe-list/recipe-list.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { RecipeService } from '../services/recipe.service';
+import { Router } from '@angular/router';
+import { recipe } from '../recipe.model';
+
+interface Recipe {
+  name: string;
+  directions: string;
+  ingredients: string;
```

Comment:
I am going to approve the request so we can keep working on this. 

In general, it makes sense to have directions and ingredients as arrays to me because they are lists of things. List of directions step by step. Lists of ingredients. 

I think the ingredients will be an array of objects. 
ingredients: array;
ingredient = {
name: string,
type: string, // from a dropdown list 
amount: number,
units: string //cups, tbs etc. 
}